### ISSUE TITLE
Sluggish MQTT feedback

### ISSUE THREAD

**Issue by kas2** [Edzelf/ESP32-Radio#107]
Ed
It takes up to 10 seconds to get MQTT feedback,
not very convenient for volume adjustment (no TFT option)

Do you foresee any problem to replace
if ( ( millis() - pubtime ) < 10000 )            // 10 seconds
by
if ( ( millis() - pubtime ) < 1000 )            // 1 second

This comment is not really an "issue", is there a more suitable place (forum, blog) for this type of discussion??
TIA

**Comment by Edzelf** [comment]
Yes, you may try shorter publish interval.  But it may influence the quality of the input stream.
